.gitignore

```diff
@@ -51,17 +50,12 @@
 .idea/**/uiDesigner.xml
 .idea/**/dbnavigator.xml
 
-# Gradle
+# IntelliJ IDEA with Gradle or Maven
+# See : https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems
 .idea/**/gradle.xml
-.idea/**/libraries
-
-# Gradle and Maven with auto-import
-# When using Gradle or Maven with auto-import, you should exclude module files,
-# since they will be recreated, and may cause churn.  Uncomment if using
-# auto-import.
-# .idea/modules.xml
-# .idea/*.iml
-# .idea/modules
+.idea/libraries/
+.idea/modules.xml
+*.iml
```

Comment:
[INFO] Move from line 18 because a support page says “you need to share: ... All the **.iml** module files” and “You may consider not to share the following: **.iml** files and .idea/modules.xml file for the Gradle or Maven based projects”